honeybee_radiance_folder/folder.py

```diff
@@ -730,6 +734,82 @@ def octree_scene_mapping(self):
 
         return scene_mapping
 
+    def grid_mapping(self):
+        """List of grids for each light path. The light paths are grouped as two phase,
+        three phase, and five phase. Two phase consist of static apertures and aperture
+        groups without a tmtx key in their states. The rest, aperture groups with a tmtx
+        key in their states, are added to three phase. Five phase is a copy of three
+        phase."""
+
+        two_phase, three_phase = [], []
+
+        grid_info = self.grid_info()
+
+        states = self.aperture_groups_states(full=True)
+        mtx_groups = []
+        non_mtx_groups = []
+        # get list of mtx groups and non-mtx groups
+        for aperture_group, ap_states in states.items():
+            for state in ap_states:
+                if aperture_group in mtx_groups or aperture_group in non_mtx_groups:
+                    continue
+                if 'tmtx' in state:
+                    mtx_groups.append(aperture_group)
+                else:
+                    non_mtx_groups.append(aperture_group)
+
+        two_phase_dict = dict()
+        three_phase_dict = dict()
+
+        for grid in grid_info:
+            light_paths = grid['light_path']
+            for light_path in light_paths:
+                light_path = light_path[0]
```

Comment:
I would rename this to make it easier to understand.

```suggestion
                first_aperture = light_path[0]
```

Reply:
The reason why I take the first index is because `light_paths` is formatted as below:
`[['static_apertures'], ['aperture_1'], ['aperture_2'], ...]`

So it's not really the first aperture, it's simply because the light paths comes as a list of lists.